### DISCUSSION TITLE
How to display birthdays before or after today

### DISCUSSION THREAD

**Discussion by MikeWILA** [microsoft-search/pnp-modern-search#3812] [Q&A]
Hi, 
We've implemented the pnp-search results as a tool to display birthdays, using the instructions from the following guide:
https://github.com/microsoft-search/pnp-modern-search/blob/main/docs/scenarios/Setup-Results-web-part-to-show-birthdays.md

In this guide, using the following query will display birthdays occuring today: 
RefinableDate00=2000-{CurrentMonth}-{CurrentDate}T00:00:00Z

However, when birthdays occur in the weekend, nobody will see this. Therefore, we want to also display birthdays that happen one/two days in the future. How can I do that?

**Comment by wobba** [comment]
I think we would need a new veriable to accommodate this as you need to subtract/add to the month if the day roll over to produce a range query. Something like `{Date(2000-01-01)}` which can take a date as input and then do math on it - similar to what we have as helpers in the results https://microsoft-search.github.io/pnp-modern-search/extensibility/templating/#getdate

**Comment by MikeWILA** [comment]
Is this something that we can do or is there a code-change needed for that? And how would I do that?

**Reply by wobba** [reply]
Code change needed, and preferably you/someone return the work as a pull request to the project :)
